src/main/webapp/main.js

```diff
@@ -80,8 +81,6 @@ connection.onmessage = (event) => {
  * @return {string} The server's URL.
   */
 function getServerUrl() {
-    
-    var defaultChatRoomID = "1234goroom";
     var protoSpec;
     var defaultIDToken = 12;
```

Comment:
What is it used for?

Reply:
The method is used to return the URL of a chat room server in a format that could be accessed by a WebSocket.